.circleci/config.yml

```diff
@@ -24,6 +44,9 @@ jobs:
       - run: npm test
 workflows:
   version: 2
-  build:
+  lint_and_build:
```

Comment:
Hmm. Does it need to be two workflows in order to show up separately in the PR?

My guess is you may have to fix some linting issues as part of this PR too

Reply:
I think we need two workflows. 

And ah... I thought so but I was afraid to change it hahah.
This is my first hacktober fest so...

Thanks a lot for the feedback

Best,
Vivek

Reply:
No I don't think we need two workflows, because build job already depends on a successful lint.

And I checked it too, it does show up seperately.

![Untitled](https://user-images.githubusercontent.com/32357540/95022914-94ba5580-0697-11eb-95d8-45d3f69b12d9.png)